exe/getinline

```diff
@@ -5,21 +5,80 @@ $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'getinline'
 require 'optparse'
 
-options = {}
+options = {
+  :mode => :html
+}
+premailer_options = {
+  :base_url => nil,
+  :link_query_string => nil,
+  :remove_classes => false,
+  :verbose => false,
+  :line_length => 65,
+  :warn_level => Premailer::Warnings::SAFE
+}
```

Comment:
Personal problem: Doesn't have to happen now, but can we change these to colons instead of hash rockets? I'm fine with that being a #techdebt ticket.